documentation/asciidoc/stories/assembly_setting_up_xsite.adoc

```diff
@@ -2,18 +2,44 @@ ifdef::context[:parent-context: {context}]
 [id='setting-up-xsite']
 :context: cross-site
 = Setting up cross-site replication
-Cross-site replication allows you to back up data from one {brandname} cluster to another.
+Ensure service availability, and guarantee business continuity, with {ispn_operator} by configuring cross-site replication to back up data between {brandname} clusters.
 
-To set up cross-site replication, you configure {ispn_operator} to automatically discover and manage connections between {brandname} clusters or manually specify the network locations for backup locations.
+//Automatically forming cross-site views
+include::{topics}/con_xsite_replication.adoc[leveloffset=+1]
+//
+//Community content
+//
+ifdef::community[]
+=== {k8s} clusters
+Apply cluster roles and then create site access secrets if you run {ispn_operator} on vanilla {k8s} or minikube.
 
-[NOTE]
-====
-You can use both automatic and manual connections for {brandname} clusters in the same `Infinispan` CR.
-However, you must ensure that {brandname} clusters establish connections in the same way at each site.
-====
+include::{topics}/proc_applying_cluster_roles_xsite.adoc[leveloffset=+3]
+include::{topics}/proc_creating_kubernetes_site_secrets.adoc[leveloffset=+3]
 
-include::assembly_automatically_connecting_clusters.adoc[leveloffset=+1]
-include::assembly_manually_connecting_clusters.adoc[leveloffset=+1]
+=== {openshiftshort} clusters
+Create and exchange service account tokens if you run {ispn_operator} on vanilla {k8s} or minikube.
```

Comment:
Don't you have to exchange tokens on openshift as well?

Reply:
whoops. typo! thanks for catching this. it should be for OpenShift only.